sim/common/sod/items_sets/phase_3.go

```diff
@@ -11,27 +11,71 @@ import (
 //                                 Cloth
 ///////////////////////////////////////////////////////////////////////////
 
-// TODO: New Set Bonuses
 var ItemSetMalevolentProphetsVestments = core.NewItemSet(core.ItemSet{
 	Name: "Malevolent Prophet's Vestments",
 	Bonuses: map[int32]core.ApplyEffect{
 		2: func(agent core.Agent) {
-			// c := agent.GetCharacter()
+			c := agent.GetCharacter()
+			c.AddStat(stats.MeleeCrit, 1)
+			c.AddStat(stats.SpellCrit, 1)
```

Comment:
Personally I feel like we should keep both set bonuses inactive if both aren't implemented so that users don't see stat changes and think the full set is implemented. Curious what you think though